.github/workflows/docs-latest.yaml

```diff
@@ -34,6 +34,11 @@ jobs:
         run: |
           sed -i 's/^Sitemap: /Sitemap: https:\/\/docs.lakefs.io/' robots.txt
 
+      - name: Add base-url to sitemap.xml
```

Comment:
Will this run for all versions or just the latest? What happens when the latest becomes the not-so-latest? Won't it continue to point to the latest link?

Reply:
We have two actions: `docs-latest.yaml` and `docs-release.yaml`.

The first is dispatched upon push-to-master, and builds and pushes the default docs (meaning, whatever is in `docs/_site`).
The second is dispatched upon release ("publish"), and builds and pushes the docs for the specific released version, into the version's folder (meaning, into `docs/_site/v1.31/`).

So this new step will run for every merge to master, and will override the existing sitemap.

(But that's a good point - changing the PR description according to this.)